crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -364,25 +371,37 @@ impl Iterator for LiveSetIter<'_> {
 
 // These functions are used to initialize the DB tables
 fn owned_object_transaction_locks_table_default_config() -> DBOptions {
-    point_lookup_db_options()
+    optimized_for_high_throughput_options(
+        read_size_from_env(ENV_VAR_LOCKS_BLOCK_CACHE_SIZE).unwrap_or(1024),
+        false,
+    )
 }
 
 fn objects_table_default_config() -> DBOptions {
-    let db_options = point_lookup_db_options();
     DBOptions {
-        options: db_options.options,
+        options: optimized_for_high_throughput_options(
+            read_size_from_env(ENV_VAR_OBJECTS_BLOCK_CACHE_SIZE).unwrap_or(5 * 1024),
+            false,
+        )
+        .options,
         rw_options: ReadWriteOptions {
             ignore_range_deletions: true,
         },
     }
 }
 
 fn transactions_table_default_config() -> DBOptions {
-    point_lookup_db_options()
+    optimized_for_high_throughput_options(
+        read_size_from_env(ENV_VAR_TRANSACTIONS_BLOCK_CACHE_SIZE).unwrap_or(512),
+        true,
+    )
 }
 
 fn effects_table_default_config() -> DBOptions {
-    point_lookup_db_options()
+    optimized_for_high_throughput_options(
```

Comment:
i think this function currently only applies to `effects` (used for state sync), not `executed_effects` (written to after execution).

We may also want to apply this to the events table as well.

Reply:
Yeah, I think `executed_effects: DBMap<TransactionDigest, TransactionEffectsDigest>` being used for storing digest is much smaller as compared in size to `effects`. So, it is used heavily but doesn't grow as much is size. I'll add this to `events` though